### PR TITLE
New designs for notifications & rename ChecNotification to ChecToast

### PR DESCRIPTION
**Pull request by dvnkshl**
image

### CODE REVIEW

File: src/components/ChecToast.vue [hunk 1]
```diff
@@ -16,16 +25,18 @@
...
   components: {
     SvgCloseIcon,
+    ChecIcon,
   },
   props: {
     /**
-     * The style variant for the notification. One of "success", "error", "warning", "info". Default is "info"
-     */
+         * The style variant for the notification. One of "success", "error", "warning", "info". Default is "info"
+         */
```
Comment (ScopeyNZ) [inline src/components/ChecToast.vue:39]: Your IDE is doing something strange here I'm guessing? These comments were correct before.

File: src/components/ChecToast.vue [hunk 2]
```diff
@@ -34,17 +45,27 @@ export default {
...
+         * The time in milliseconds after which a notification will disappear, if `persist` is true
+         */
     hideTime: {
       type: Number,
       default: 5000, // milliseconds
     },
   },
+  data() {
+    return {
+      icon: {
```
Comment (ScopeyNZ) [inline src/components/ChecToast.vue:61]: This is a constant and shouldn't be defined in `data`. Variables created here will be reactive, which is unnecessary for this.

**Review by ScopeyNZ — CHANGES_REQUESTED** [review]
I'll have a look at the deploy preview in a moment.

File: src/components/ChecToast.vue (hunk 2 above)
Comment (ScopeyNZ) [reply src/components/ChecToast.vue:61]: How I'd recommend you change this is just create a `computed` that does something like:

```
const icons = {...};
return icons[this.variant];
```

File: src/components/ChecToast.vue (hunk 1 above)
Comment (dvnkshl) [reply src/components/ChecToast.vue:39]: yeah weird fixed - locally

**Comment by ScopeyNZ** [comment]
The hover state on the button is broken: 

image

The button style is wrong, as there seems to be some font styling that's overwriting the button font, but I guess that this is a pre-existing issue.

**Comment by dvnkshl** [comment]
@ScopeyNZ updated

File: src/components/ChecToast.vue [hunk 3]
```diff
@@ -80,29 +101,36 @@ export default {
...
-      @apply mt-2;
+      @apply mt-3 -ml-1;
     }
+
   }
 
   &__close-icon {
-    @apply ml-4 p-2 bg-green-600 rounded text-white cursor-pointer;
+    @apply bg-green-600 rounded text-white cursor-pointer mr-2 mt-4 p-2;
+    transition: all 0.3s ease, outline 0ms !important;
```
Comment (ScopeyNZ) [inline src/components/ChecToast.vue:131]: Why do we have an `!important` here?

File: src/components/ChecToast.vue [hunk 4]
```diff
@@ -80,29 +101,36 @@ export default {
...
 
-    > svg {
+    >svg {
```
Comment (ScopeyNZ) [inline src/components/ChecToast.vue:133]: This was better the way it was before...

File: src/components/ChecToast.vue (hunk 3 above)
Comment (dvnkshl) [reply src/components/ChecToast.vue:131]: Without !important

 

With !important

Comment (ScopeyNZ) [reply src/components/ChecToast.vue:131]: Right, if you want to get rid of the outline, just make sure it's `outline: none`.

**Review by ScopeyNZ — APPROVED** [review]

File: src/stories/components/ChecNotification.stories.mdx [hunk 5]
```diff
@@ -2,19 +2,19 @@ import { Meta, Props, Story, Preview } from '@storybook/addon-docs/blocks';
 import { action } from "@storybook/addon-actions";
 import { text, select } from '@storybook/addon-knobs';
 import ChecButton from '../../components/ChecButton.vue';
-import ChecNotification from '../../components/ChecNotification.vue';
+import ChecToast from '../../components/ChecToast.vue';
 
-<Meta title="Components/Notifications" component={ChecNotification} />
+<Meta title="Components/Toast " component={ChecToast} />
```
Comment (robbieaverill) [inline src/stories/components/ChecNotification.stories.mdx:7]: Next time you're in here please rename the file as well

**Pull request merged**